app/src/src/contexts/UserViewContext.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState, createContext } from 'react';
+
+const UserViewContext = createContext();
+
+// A context provider that stores the user whose profile is to be shown
```

Comment:
Follow the same commenting structure and coding practices as the other Contexts.